.github/workflows/publish_docs.yml

```diff
@@ -23,13 +23,10 @@ jobs:
           git config user.email github-actions@github.com
 
       - name: Generate documentation
-        run: bundle exec rake rdoc
-
-      - name: Copy README images to the docs folder
+        working-directory: ./jekyll
         run: |
-          mkdir docs/vscode
-          cp vscode/icon.png docs/vscode/icon.png
-          cp -R vscode/extras docs/vscode/
+          bundle install
```

Comment:
This is never going to cache dependencies. Can we use [working-directory](https://github.com/ruby/setup-ruby?tab=readme-ov-file#working-directory) from the setup-ruby action?